### PR TITLE
serializers : geojson maping + testing

### PR DESCRIPTION
**Pull request by TLGINO**
* can now export to geojson (for Points)
* closes https://github.com/inveniosoftware/invenio-rdm-records/issues/1154

### CODE REVIEW

File: invenio_rdm_records/services/services.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def search_community_records(
         self.require_permission(identity, "read")
         current_communities.service.record_cls.pid.resolve(
             community_id
-        )  # Checks weather community exists
+        )  # Checks wether community exists
```
Comment (kpsherva) [inline invenio_rdm_records/services/services.py:137]: I think we probably need `whether` here

File: invenio_rdm_records/resources/serializers/geojson/schema.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from marshmallow import Schema, fields
+
+
+class GeoJSONSchema(Schema):
+    """Schema for GeoJSON in JSON."""
+
+    features = fields.Method("get_locations")
+    type = fields.Constant(
+        "FeatureCollection", dump_to="type"
+    )  # Will always return Feature Collection, eventhough there may be just 1 point
```
Comment (kpsherva) [inline invenio_rdm_records/resources/serializers/geojson/schema.py:19]:
```suggestion
    )  # Will always return Feature Collection, even though there may be just 1 point
```

File: invenio_rdm_records/resources/serializers/geojson/schema.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
```
Comment (kpsherva) [inline invenio_rdm_records/resources/serializers/geojson/schema.py:3]:
```suggestion
# Copyright (C) 2023 CERN

File: invenio_rdm_records/resources/serializers/geojson/__init__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
```
Comment (kpsherva) [inline invenio_rdm_records/resources/serializers/geojson/__init__.py:3]:
```suggestion
# Copyright (C) 2023 CERN
```

File: invenio_rdm_records/resources/serializers/geojson/schema.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+        for l in obj["metadata"]["locations"]["features"]:
+            item = {
+                "type": "Feature",
+                "geometry": {
+                    "type": l["geometry"]["type"],
+                    "coordinates": l["geometry"]["coordinates"],
+                },
+                "properties": {
+                    a: l[a] for a in l if a != "geometry"
```
Comment (kpsherva) [inline invenio_rdm_records/resources/serializers/geojson/schema.py:36]: does it make sense to give the variables some more self-documenting names ?

**Review by ntarocco — COMMENTED** [review]
LGTM! Thanks!

File: tests/resources/serializers/test_geojson_serializer.py [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+                            {"scheme": "geonames", "identifier": "2661235"}
+                        ],
+                        "place": "CERN",
+                        "description": "Invenio birth place.",
+                    },
+                    {
+                        "geometry": {"type": "Point", "coordinates": [12, 34]},
+                        "identifiers": [{"scheme": "geonames", "identifier": "111"}],
+                        "place": "TESTING",
+                        "description": "AHHHHHHHHHHHHHHHHHHHHHHH",
```
Comment (zzacharo) [inline tests/resources/serializers/test_geojson_serializer.py:124]: Maybe a more meaningful description? 😅

**Review by zzacharo — APPROVED** [review]
Nice work!

**Pull request merged**